packages/lab/src/dialog/Dialog.tsx

```diff
@@ -89,49 +104,46 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(function Dialog(
     [dialogId, status]
   );
 
+  if (!showComponent) {
+    return <></>;
+  }
+
   return (
-    <FloatingPortal>
-      {/* The provider is needed to support the use case where an app has nested modes. The element that is portalled needs to have the same style as the current scope */}
-      <SaltProvider>
-        {showComponent && (
-          <FloatingOverlay className={withBaseName("overlay")} lockScroll>
-            <FloatingFocusManager
-              context={context}
-              modal
-              initialFocus={initialFocus}
-            >
-              <DialogContext.Provider value={contextValue}>
-                <div
-                  id={dialogId}
-                  className={clsx(
-                    withBaseName(),
-                    {
-                      [withBaseName("enterAnimation")]: open,
-                      [withBaseName("exitAnimation")]: !open,
-                      [withBaseName("withStatus")]: status,
-                      [withBaseName(status as string)]: status,
-                    },
-                    className
-                  )}
-                  onAnimationEnd={() => {
-                    if (!open && showComponent) {
-                      setShowComponent(false);
-                    }
-                  }}
-                  ref={floatingRef}
-                  aria-labelledby={`${dialogId}-heading`}
-                  aria-describedby={`${dialogId}-description`}
-                  aria-modal="true"
-                  {...getFloatingProps()}
-                  {...rest}
-                >
-                  {children}
-                </div>
-              </DialogContext.Provider>
-            </FloatingFocusManager>
-          </FloatingOverlay>
-        )}
-      </SaltProvider>
-    </FloatingPortal>
+    <Scrim>
+      <DialogContext.Provider value={contextValue}>
+        <FloatingComponent
+          id={`${dialogId}-FLoatigComponent`}
+          open={open}
+          aria-modal="true"
+          aria-labelledby={`${dialogId}-header`}
+          aria-describedby={`${dialogId}-description`}
```

Comment:
Is aria-labelledby / describedby only needed on the parent component, of should title and content also have these aria labels ?